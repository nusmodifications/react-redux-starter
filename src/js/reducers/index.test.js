// @flow
import type { FSA } from 'types/redux';

import rootReducer from 'reducers';

describe('rootReducer', () => {
  it('has all the child reducers', () => {
    const initialState = {};
    const dummyAction: FSA = {
      type: 'DUMMY_ACTION',
      payload: null,
    };
    const state = rootReducer(initialState, dummyAction);
    expect(state).toMatchSnapshot();
  });
});
