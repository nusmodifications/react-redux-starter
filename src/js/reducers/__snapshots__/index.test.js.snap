// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rootReducer has all the child reducers 1`] = `
Object {
  "counter": 0,
  "errors": Object {},
  "reddits": Array [],
  "requests": Object {},
}
`;
